scripts/DGSPlanner/DGSPlannerGUI.py

```diff
@@ -19,6 +19,8 @@
 from matplotlib.figure import Figure
 from mpl_toolkits.axisartist.grid_helper_curvelinear import GridHelperCurveLinear
 from mpl_toolkits.axisartist import Subplot
+from scipy.constants import m_n
+from scipy.constants import h
```

Comment:
You can use the constants from mantid instead of a third-party library. `mantid.kernel.PhysicalConstants.h` and `mantid.kernel.PhysicalConstants.NeutronMass`.